pkg/cmd/create.go

```diff
@@ -86,6 +88,11 @@ func createAction(_ *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to get config: %w", err)
 	}
 
+	_, err = providerplugin.CheckProviderExecutable(config)
```

Comment:
this should be only done in case the node-images/ directory exist right?

Reply:
@janiskemper yes, the short-cut to skip the step is not implemented yet. Maybe we should check node-images.yaml instead of the directory? Imagine aws/gcp just need a bit of config, and one yaml file would be enough for them. Then these providers don't need a node-images directory. What do you think?

Reply:
@janiskemper I updated the implementation. The file csmctl.yaml is now the source of truth. If there is no "config" (below provider), then the plugin does not get called.